tests/de/gurkenlabs/litiengine/environment/tilemap/xml/BooleanIntegerAdapterTests.java

```diff
@@ -0,0 +1,43 @@
+package de.gurkenlabs.litiengine.environment.tilemap.xml;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
```

Comment:
Please remove the unused import `org.junit.jupiter.api.Assertions.assertFalse;`

Reply:
removed 👍